dash/development/component_generator.py

```diff
@@ -58,8 +58,11 @@ def generate_components(
     reserved_patterns = "|".join("^{}$".format(p) for p in reserved_words)
 
     os.environ["NODE_PATH"] = "node_modules"
+
+    fmt = "node {} \"{}\" \"{}\" {}" if is_windows else "node {} {} {} {}"
```

Comment:
Thanks @stevej2608 !
Is it necessary that this change is *only* made on Windows? Can't we just *always* quote these arguments? Obviously we (Plotly devs) don't build on Windows very often, so I worry it'll just break again if Windows has special code.